docs/README.md

```diff
@@ -181,6 +181,22 @@ Sub-commands:
 * `-f`, `--force`: Overwrite existing scenario setup file.
 * `-h`, `--help`: Show the help for the `template` command.
 
+##### Piping stdin
+
+Config file and overrides file can be piped from stdin. If no config is set stdin is assumed to be the config file, if config file is set, stdin is assumed to be the overrides file.
+
+This would execute the sheetchanger example from stdin:
+
+```bash
+cat ./docs/examples/sheetChangerQlikCore.json | ./gopherciser x
+```
+
+This would execute overrides from stdin:
+
+```bash
+cat overrides.txt |./gopherciser x -c ./docs/examples/sheetChangerQlikCore.json
```

Comment:
nit: add a space
```suggestion
cat overrides.txt | ./gopherciser x -c ./docs/examples/sheetChangerQlikCore.json
```

Reply:
why?

Reply:
fwiw I agree with Anton

Reply:
semantically the same, but it is the norm to surround with spaces from what I know

Reply:
really? my term emulator always automatically remove the space from there